hetio/neo4j.py

```diff
@@ -71,7 +101,13 @@ def append(self, x):
     def create(self):
         if not self:
             return
-        self.db_graph.create(*self)
+
+# http://stackoverflow.com/questions/37676731/typeerror-create-takes-2-positional-arguments-but-4-were-given
+        if PY2NEO_VER == 3:
```

Comment:
Perhaps `if PY2NEO_VER >= 3` makes a little more sense, assuming v4 would be more like 3 than 2